scripts/convert_checkpoint.py

```diff
@@ -92,6 +92,9 @@ def meta_weights_for_nano_model(
     checkpoint_files.sort()
     n_checkpoints = len(checkpoint_files)
 
+    if n_checkpoints == 0:
+        raise RuntimeError(f"No checkpoints were found at ckpt_dir ${ckpt_dir}. `consolidated.0*.pth` files expected at that location.")
```

Comment:
is the dollar sign intentional or a typo?

Reply:
typo, thanks!